### PR TITLE
Feature view tasks for particular day

### PR DESCRIPTION
**Pull request by Ramanathan0908**
This PR adds the tasksOn command which lists the tasks on a particular day.
It's still a work in progress as I have not wrote the test cases yet, but the functionality of command should be there

### CODE REVIEW

File: src/main/java/seedu/uninurse/model/task/TaskList.java [hunk 1]
```diff
@@ -95,13 +103,13 @@ public boolean isEmpty() {
     /**
      * Returns the internal task list.
      */
-    public ArrayList<Task> getTasks() {
-        return internalTaskList;
+    public List<Task> getTasks() {
+        return filteredTaskLists;
     }
```
Comment (Ramanathan0908) [inline src/main/java/seedu/uninurse/model/task/TaskList.java:108]: I have changed the return type to List<Tasks>, but decided to keep the method getTasks() and instead use it to return filteredTaskList, specific to TaskList

File: docs/UserGuide.md [hunk 2]
```diff
@@ -191,6 +191,17 @@ Shows a list of all patients with tasks due today
 
 Format: `patientsToday`
 
+### Listing all tasks for a particular day: `tasksOn`
+
+Shows a list of all tasks on a particular day
```
Comment (izzahaj) [inline docs/UserGuide.md:196]: Missed a full stop.
```suggestion
Shows a list of all tasks on a particular day.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -191,6 +191,17 @@ Shows a list of all patients with tasks due today
...
+
+Format: `tasksOn DATE`
+
+* The DATE **must be of the specified format** dd-MM-yyyy
+
+Examples:
+* `tasksOn 25-12-2022` lists the tasks on 25th december 2022
```
Comment (izzahaj) [inline docs/UserGuide.md:203]: Months should start with an uppercase letter.
```suggestion
* `tasksOn 25-12-2022` lists the tasks on 25th December 2022
```

P

**Review by izzahaj — CHANGES_REQUESTED** [review]
LGTM except for very minor issues in the user guide. There are also merge conflicts, so I suggest resolving those first.

**Review by izzahaj — APPROVED** [review]
LGTM

**Pull request merged**